src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableList_1.DebuggerProxy.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+
+namespace System.Collections.Immutable
+{
+    /// <summary>
+    /// A simple view of the immutable list that the debugger can show to the developer.
+    /// </summary>
+    internal class ImmutableListDebuggerProxy<T>
```

Comment:
Self-note: This will have to go once the PR with the consolidated proxies goes in